include/battle_util.h

```diff
@@ -325,7 +327,7 @@ u32 GetBattlerType(u32 battler, u32 typeIndex, bool32 ignoreTera);
 bool8 CanMonParticipateInSkyBattle(struct Pokemon *mon);
 bool8 IsMonBannedFromSkyBattles(u16 species);
 void RemoveBattlerType(u32 battler, u8 type);
-u32 GetMoveType(u32 move);
+u32 GetBattleMoveType(u32 move);
```

Comment:
You changed `GetMoveType` to `GetBattleMoveType` but isn't the old `GetMoveType` already a form of encapsulation? So you are doubly encapsulating it?

Reply:
This PR's `GetMoveType` is done specifically to accessing the field in gMovesInfo (which, outside of tests, is used dozens of times. See: https://github.com/rh-hideout/pokeemerald-expansion/pull/5852/commits/d2c730cbaaeb7eb090a752b3a553204ffc038c79

`GetBattleMoveType`'s rename is meant to represent its actual use in batlles, as it reads from dynamic move data. I could unify it if you want to make it cleaner :)

Reply:
Yeah, Ig the question is why we do the encapsulation here in the first place. If it should be one entry point to data where additional things can be added then I think it would make sense to combine them. I would say the same applies to priority. 

We could maybe function it out inside the get function so they are visually split from each other and a user has an easier time adding things to them